pages/api/plants/index.js

```diff
@@ -7,6 +7,19 @@ export default async function handler(request, response) {
   if (request.method === "GET") {
     const plants = await Plant.find();
     return response.status(200).json(plants);
+  }
+  if (request.method === "POST") {
+    try {
+      const plantData = request.body;
+      const newPlant = await Plant.create(plantData);
+      response
+        .status(201)
+        .json({ message: "New crop successfully created", id: newPlant._id });
+      console.log(newPlant);
+    } catch (error) {
+      console.log(error);
```

Comment:
this console log I would consider valid though

Reply:
We've removed it as well, as not needed for us and `response.status` is already logged.